src/main/java/us/kbase/auth/AuthService.java

```diff
@@ -0,0 +1,25 @@
+package us.kbase.auth;
+
+import java.io.IOException;
+
+// TODO CODE remove if JsonClientCaller in java comment gets rewritten to remove this.
+
+/** This class is here only to trick extremely crufty code and serves no useful purpose. */
+@Deprecated
+public class AuthService {
+	
+	/** Throws {@link UnsupportedOperationException}.
+	 * @param userName ignore me
+	 * @param password ignore me
+	 * @return nothing useful
+	 * @throws AuthException never
+	 * @throws IOException never
+	 */
+	public static AuthUser login(
```

Comment:
Are there things that use this? Do we need to keep supporting it in new code?

Reply:
Is there an `@obsoleted` decorator or something you can use? `Deprecated` implies it still kinda works, but won't in the future, and this just always fails.

Reply:
> Are there things that use this?

JsonClientCaller in java common

Reply:
> Is there an `@obsoleted` decorator or something you can use?

Not that I'm aware of

https://docs.oracle.com/javase/tutorial/java/annotations/predefined.html